MultiPartyExample/LocalParticipantView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  LocalParticipantView.swift
+//  MultiPartyExample
+//
+//  Created by Ryan Payne on 4/17/19.
+//  Copyright © 2019 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+import TwilioVideo
+
+@IBDesignable
+class LocalParticipantView: UIView {
+
+    @IBOutlet var contentView: UIView!
+    @IBOutlet weak var videoView: TVIVideoView!
+    @IBOutlet weak var noVideoImage: UIImageView!
+    @IBOutlet weak var audioIndicator: UIImageView!
+    @IBOutlet weak var networkQualityLevelContainerView: UIView!
+    @IBOutlet weak var networkQualityLevelIndicator: UIImageView!
+
+    var recognizerDoubleTap: UITapGestureRecognizer?
+
+    var hasAudio: Bool = false {
+        willSet {
+            audioIndicator.isHidden = newValue
+        }
+    }
+
+    var hasVideo: Bool = false {
+        willSet {
+            videoView.isHidden = !newValue
+            videoView.alpha = newValue ? 1.0 : 0.0
+            noVideoImage.isHidden = newValue
+        }
+    }
+
+    var networkQualityLevel: TVINetworkQualityLevel = .unknown {
+        willSet {
+            let info = networkQualityLevelIndicatorInfo(newValue)
+
+            guard let networkQualityLevelImage = info.networkQualityLevelImage,
+                let tintColor = info.tintColor else {
+                    networkQualityLevelContainerView.isHidden = true
+                    return
+            }
+
+            networkQualityLevelContainerView.isHidden = false
+            networkQualityLevelIndicator.image = networkQualityLevelImage
+            networkQualityLevelIndicator.tintColor = tintColor
+        }
+    }
+
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    private func setup() {
+        Bundle.main.loadNibNamed("LocalParticipantView", owner: self, options: nil)
+        addSubview(contentView)
+        contentView.frame = self.bounds
+        contentView.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+
+        audioIndicator.layer.cornerRadius = audioIndicator.bounds.size.width / 2.0;
+        audioIndicator.layer.backgroundColor = UIColor.black.withAlphaComponent(0.75).cgColor
+
+        noVideoImage.isHidden = false
+        networkQualityLevelContainerView.isHidden = true
+        networkQualityLevelContainerView.layer.backgroundColor = UIColor.black.withAlphaComponent(0.75).cgColor
```

Comment:
From a performance standpoint we shouldn't be performing two rounds of alpha blending (one for the transparent UIImage, and one for this UIView container) on top of live video content. I think its okay for this PR, but I would add a comment mentioning how it could be improved.

Reply:
Add a comment for this?